CHANGES.rst

```diff
@@ -68,6 +68,8 @@ astropy.table
 - Improved exception handling and error messages when column ``format`` 
   attribute is incorrect for the column type. [#6385]
 
+- Added new table operation ``astropy.table.setdiff``. [#6443]
```

Comment:
As long as there will be one more set of changes, can you add a little more description to this?  E.g. 
```
Added a new table operation ``astropy.table.setdiff`` that returns the set difference
of table rows for two tables. [#6443]
```